DTS-STN/Service-Canada-Design-System
# Authenticated header to have dynamic menu items

https://dev.azure.com/VP-BD/DECD/_sprints/taskboard/Secure%20Client%20Hub/DECD/Secure%20Client%20Hub/SCH%20Beta%20Sprint%204?workitem=73175

#### Description & Background Context:
Changed authenticated menu to use a dynamic list based on provided props
Added test ids to header and footer to test rendering

#### Where should the reviewer start?
Check header, footer and menu components in storybooks

#### How should this be tested?
Change menu props within the header and menu. Ensure the menu displays and functions correctly

#### Are there any breaking changes?
There are a significant amount of failing tests. I see 90 failing in develop and 90 failing on this branch so I assume those are ongoing issues and unrelated.
